setup/README.md

```diff
@@ -16,15 +16,15 @@ A guide to setting up your Raspberry Pi
     - For keyboard layout configuration options see [raspi-config](../configuration/raspi-config.md).
 - [Power supply](../hardware/raspberrypi/power/README.md)
     - The Pi is powered by a USB Micro power supply (like most standard mobile phone chargers).
-    - You'll need a good-quality power supply that can supply at least 2A at 5V for the Model 3B, or 700mA at 5V for the earlier, lower powered models.
+    - You'll need a good-quality power supply that can supply at least 2A at 5V for the Model 3B and 3B+, or 700mA at 5V for the earlier, lower powered models. We recommend the official [Raspberry Pi power supply](https://www.raspberrypi.org/products/raspberry-pi-universal-power-supply/), which is designed specifically for all Raspberry Pi models.
     - Low current (~700mA) power supplies will work for basic usage, but are likely to cause the Pi to reboot if it draws too much power.
 
 ### Optional
 
 - Ethernet (network) cable [Model B/B+/2/3 only]
```

Comment:
Does this line also need updating, given that the Model 3A+ doesn't have Ethernet?

Reply:
Yup